util/metrics/src/service_metrics.rs

```diff
@@ -170,9 +197,6 @@ fn path_from_ctx(ctx: &RpcContext) -> Option<String> {
 /// `/{package}.{service_name}/{method}`
 /// and converts it into a dot-delimited string, dropping the 1st `/`
 fn path_from_byte_slice(bytes: &[u8]) -> Option<String> {
-    // The content of method() is expected to be:
-    // `/{package}.{service}/{method}`
-    // 47 ascii = '/'
```

Comment:
nit: is there a good reason to remove this comment?

Reply:
De-duping. `/{package}.{service}/{method}` is duplicated in the method docstring above. The // 47 ascii specification could be moved up to the docstring or left where it is. Thoughts?

Reply:
The 47 comment seems good

I think we didn't actually write this code, we lifted it from some other company. Don't remember exactly